files/borgback-adfc-final.sh

```diff
@@ -0,0 +1,116 @@
+# Backup-Skript mit Prüfung auf aktive Verbindung zu den Backup-Satelliten
+# Ver. 0.7 vom 30.11.2019, Autor: Volker Kraeft [post@volkerkraeft.de]
+
+# ---------- Variablen-Definition ---------------------------------------------------------
+
+TIMESTAMP=$(date +"%Y%m%d-%H%M%S")                   # Zeitstempel für Logdatei und Backuparchiv
+LOG_DIR="/pfad/zum/logverz"                          # Verzeichnis für Logdaten
+LOG_DATA="borgbackup_${TIMESTAMP}.log"               # Logdatei für aktuelle Sicherung
+LOGFILE="$LOG_DIR/$LOG_DATA"                         # Zur besserern Darstellung im Code ... / FIX!
+
+CL_ONLINE_DIR="/pfad/zum/verz/bak-clients-online"    # Hier legen die Backup-Satelliten ihre Anmelde-Info ab
+
+SECRET="borgbackup-passwort"                         # Kennwort für RepoKey
+SOURCE_DIR="/pfad/zu/den/daten /datenverz"           # Zu archivierende Verzeichnisse (mit Leerzeichen getrennt)
+SOURCE_EXCL="*.tmp"                                  # Sicherungs-Ausnahmen (mit Leerzeichen getrennt)
+REPO_PATH="~/pfad/zum/repository"                    # Pfad zum Backup-Repository auf Backup-Client
+REPO_NAME="name-des-borg-repositories"               # Name des Backup-Repositories auf Backup-Client
+ARCHIVE_NAME="name-des-archivs_${TIMESTAMP}"         # Name der aktuellen Sicherung
+BORG_USER="borguser"                                 # Benutzerkonto auf Backup-Client / FIX! NICHT ÄNDERN
+
+PRUNE_D="7"                                          # Anzahl der vorzuhaltenden Tagessicherungen
+PRUNE_W="4"                                          # Anzahl der vorzuhaltenden Wochensicherungen
+PRUNE_M="12"                                         # Anzahl der vorzuhaltenden Monatssicherungen
+PRUNE_Y="2"                                          # Anzahl der vorzuhaltenden Jahressicherungen
+
+M_SND="absender@e-mail.tld"                          # Absender
+M_RCV="empfaenger@e-mail.tld"                        # Empfänger
+M_PWD="smtp-passwort"                                # SMTP-Passwort
+M_SVR="mailserver.hoster.tld"                        # SMTP-Server
+M_SUB="Backup-Report"                                # Betreffzeile / sollte so bleiben
+M_BDY="Freier Text für E-Mail-Body."                 # Body Text
+M_ATT="anhang1.ext /pfad/zum/anhang2.ext"            # weitere Anhänge hier eintragen (Backup-Logdatei geht standardmäßig immer mit raus)
+
+
+
+###########################################################################################
+############ AB HIER KEINE ANPASSUNGEN MEHR ERFORDERLICH! #################################
+###########################################################################################
+
+
+# ---------- Einlesen der verbundenen Backup-Clients / Ermitteln der IP -------------------
+
+# Ist kein Client verbunden wird auch nichts unternommen
+if [ "$(ls -A $CL_ONLINE_DIR)" ]; then
+   MACHE="WEITER"
+else
+   exit 1
+fi
+
+# Einlesen aller Verbindungen
+ALLCLIENTS=()
+for FILES in $CL_ONLINE_DIR/*; do
```

Comment:
In der Variable FILES steht nur eine Datei, bitte Namen überdenken

Reply:
$FILES heißt nun $CLIENT